src/Symfony/Bundle/TwigBundle/Extension/AssetsExtension.php

```diff
@@ -57,6 +61,8 @@ public function getFunctions()
      */
     public function getAssetUrl($path, $packageName = null, $absolute = false, $version = null)
     {
+        trigger_error('The Twig asset() function was deprecated in 2.7 and will be removed in 3.0. Please use asset_path() instead.', E_USER_DEPRECATED);
```

Comment:
I've removed this deprecation message and the one below as the whole class is now deprecated.